org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,60 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "FireflyLuciferin",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--device=all",
+    "--filesystem=xdg-documents",
+    "--filesystem=xdg-download"
+  ],
+  "modules": [
+    "shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json",
+    {
+      "name": "cuda-toolkit",
+      "only-arches": [
+        "x86_64"
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "mv lib/libnvrtc* /app/lib/"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": [
+            "x86_64"
+          ],
+          "url": "https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvrtc/linux-x86_64/cuda_nvrtc-linux-x86_64-12.6.85-archive.tar.xz",
+          "sha256": "a4289ba86f0deeccf4e1d94c58559eae4c8184fcb6eaade8fcab45584e6cf402"
+        }
+      ]
+    },
+    {
+      "name": "fireflyluciferin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x FireflyLuciferinLinux.deb",
+        "tar xvf data.tar.zst",
+        "mkdir -p /app/fireflyluciferin",
+        "echo -e '\nflatpak.env=true' >> opt/fireflyluciferin/lib/app/classes/project.properties",
+        "cp -r opt/fireflyluciferin/* /app/",
+        "install -Dm644 ./opt/fireflyluciferin/lib/app/classes/build_assets/org.dpsoftware.FireflyLuciferin.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+        "install -Dm644 ./opt/fireflyluciferin/lib/app/classes/build_assets/org.dpsoftware.FireflyLuciferin.appdata.xml /app/share/share/appdata/${FLATPAK_ID}.appdata.xml",
```

Comment:
why does the license say proprietary and then link to a GPL v3 license?

![image](https://github.com/user-attachments/assets/17d3118d-d248-4b4d-893e-14cf81b212d6)

Reply:
Raised it, apparently wasn't addressed:

https://github.com/flathub/flathub/pull/5874#discussion_r1854886259

Reply:
my bad sorry, this time, it should be really GPL v3. 
I tought you was referring to metadata_license and I misspelled the project_license.

this is the format I used now:
    <metadata_license>CC0-1.0</metadata_license>
    <project_license>GPL-3.0+</project_license>

Reply:
So where is the source code?

Reply:
I added the "vcs-browser" linking to the repo in the appdata.